tools/query_verification/src/main/java/com/google/bigquery/QueryVerifier.java

```diff
@@ -51,26 +51,38 @@ public void verifyDataFree() {
         // Create tables based on schema
         if (migratedSchema != null) {
             if (migratedSchema.isInJsonFormat()) {
-                TableInfo tableInfo = QueryVerifier.getTableInfoFromJsonSchema(migratedSchema);
-                if (tableInfo != null) {
-                    Table table = bigQuery.create(tableInfo);
-                    tables.add(table);
-                } else {
-                    System.out.println(migratedSchema.path() + " is not correctly formatted.");
+                // Schema is JSON
+                List<TableInfo> tableInfos = QueryVerifier.getTableInfoFromJsonSchema(migratedSchema);
+                if (tableInfos != null) {
+                    for (TableInfo tableInfo : tableInfos) {
+                        Table table = bigQuery.create(tableInfo);
+                        tables.add(table);
+                    }
                 }
             } else {
-                // TODO Load schema from DDL
+                // Schema is DDL
+                JobInfo jobInfo = configureJob(migratedSchema.schema(), false);
+                Job schemaJob = bigQuery.create(jobInfo);
+                try {
+                    schemaJob.waitFor();
+                } catch (InterruptedException e) {
+                    System.out.println(e.getMessage());
+                }
+
+                List<TableId> tableIds = QueryVerifier.getTableIdsFromDdlSchema(migratedSchema);
```

Comment:
Does the schema job not return information about the table created that we can use here instead of reparsing the mgiratedSchema?

Reply:
I wasn't able to find that. I think the only table information it returns are the query results, which is empty in this case.

Reply:
Testing with the BQ UI it seems like [this api](https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs/getQueryResults) is used which returns the table schema. Can you see if it's possible to use this?

Reply:
It looks like `getQueryResults` just returns null for ddl, so it doesnt return a schema for me.

Reply:
Interesting. If it doesn't work then I guess this is fine. Thanks for checking!